dokku.yml

```diff
@@ -71,6 +74,28 @@
         state: set
       with_items: "{{ apps }}"
 
+    - name: Check if DNAnalyzer postgres exists
+      shell: dokku postgres:exists dnanalyzer
+      register: pg_exists
+      changed_when: false
+      ignore_errors: true
+
+    - name: Create DNAnalyzer postgres
+      shell: dokku postgres:create dnanalyzer
+      changed_when: app_exists.rc == 1
+      when: pg_exists.rc == 1
+
+    - name: Check if DNAnalyzer postgres link exists
+      shell: dokku postgres:exists dnanalyzer
+      register: pg_link_exists
+      changed_when: false
+      failed_when: pg_link_exists.rc > 1
+
+    - name: Create DNAnalyzer postgres link
+      shell: dokku postgres:link dnanalyzer dnanalyzer
+      changed_when: "'dnanalyzer' not in pg_link_exists.stdout"
+      when: "'dnanalyzer' not in pg_link_exists.stdout"
+
```

Comment:
could you make this a loop, and loop over the values `dnanalyzer` + `gtn-slack-bot`?

Reply:
sorry, looping did not work for me directly, because of the conditionals. googled a bit but I think it's not worth the time now

Reply:
fair enough!